src/include/parser/update_statement.h

```diff
@@ -56,6 +56,10 @@ class UpdateStatement : public SQLStatement {
 
   virtual void Accept(SqlNodeVisitor *v) override { v->Visit(this); }
 
+  inline void TryBindDatabaseName(std::string default_database_name) {
+    table->TryBindDatabaseName(default_database_name);
+  }
```

Comment:
We can't add a function that is only used in the test to the source code. We have to use something else.